libs/tools/barista/src/transform.ts

```diff
@@ -202,6 +203,51 @@ export const copyHeadlineTransformer: BaPageTransformer = async source => {
   return transformed;
 };
 
+/** Replaces absolute href with contentLink */
+export const relativeUrlTransformer: BaPageTransformer = async source => {
+  const transformed = { ...source };
+  if (source.content && source.content.length) {
+    transformed.content = runWithCheerio(source.content, $ => {
+      const links = $('a');
+      if (links.length) {
```

Comment:
I don't think you need to check for length in an each statement. It will just not iterate and it will make your nesting less deep.